vimeo-networking/src/main/java/com/vimeo/networking/model/Spatial.java

```diff
@@ -0,0 +1,73 @@
+package com.vimeo.networking.model;
+
+import com.vimeo.stag.GsonAdapterKey;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.io.Serializable;
+
+/**
+ * The spatial object representing the format
+ * data for a 360 video object.
+ * <p>
+ * Created by restainoa on 1/3/17.
+ */
+public class Spatial implements Serializable {
+
+    private static final long serialVersionUID = 5660325676029549468L;
+
+    public enum Projection {
+        EQUIRECTANGULAR,
+        CYLINDRICAL,
+        CUBICAL,
+        PYRAMID,
+        DOME
+    }
+
+    public enum Format {
+        MONO,
+        LEFT_RIGHT,
+        TOP_BOTTOM
+    }
+
+    @NotNull
+    @GsonAdapterKey("projection")
+    protected String mProjection;
+
+    @NotNull
+    @GsonAdapterKey("stereo_format")
+    protected String mStereoFormat;
```

Comment:
I must have missed when Stag started allowing for protected members, I thought they had to be public of package-private. Nice to see this!